Description: cstdatomic has been renamed to atomic, fix #include statements.
Bug: http://code.google.com/p/leveldb/issues/detail?id=3
---
 build_detect_platform |    2 +-
 port/atomic_pointer.h |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: bitcoin/src/leveldb/build_detect_platform
===================================================================
--- bitcoin.orig/src/leveldb/build_detect_platform	2013-09-03 23:02:40.897471645 -0400
+++ bitcoin/src/leveldb/build_detect_platform	2013-09-03 23:03:56.625473444 -0400
@@ -189,7 +189,7 @@
 
     # If -std=c++0x works, use <cstdatomic>.  Otherwise use port_posix.h.
     $CXX $CXXFLAGS -std=c++0x -x c++ - -o $CXXOUTPUT 2>/dev/null  <<EOF
-      #include <cstdatomic>
+      #include <atomic>
       int main() {}
 EOF
     if [ "$?" = 0 ]; then
Index: bitcoin/src/leveldb/port/atomic_pointer.h
===================================================================
--- bitcoin.orig/src/leveldb/port/atomic_pointer.h	2013-09-03 23:02:40.897471645 -0400
+++ bitcoin/src/leveldb/port/atomic_pointer.h	2013-09-03 23:02:40.893471644 -0400
@@ -21,7 +21,7 @@
 
 #include <stdint.h>
 #ifdef LEVELDB_CSTDATOMIC_PRESENT
-#include <cstdatomic>
+#include <atomic>
 #endif
 #ifdef OS_WIN
 #include <windows.h>
